src/main/java/it/gov/pagopa/pu/classification/service/assessments/AssessmentsDetailServiceImpl.java

```diff
@@ -114,6 +136,14 @@ private static AssessmentsDetail buildAssessmentsDetail(ReceiptNoPII receipt, In
             .assessmentCode(assessmentCode)
             .amountCents(amountCents)
             .receiptId(receipt.getReceiptId())
+            .officeDescription(officeDescription)
+            .sectionDescription(sectionDescription)
+            .assessmentDescription(assessmentDescription)
+            .debtPositionTypeOrgId(debtPositionTypeOrgId)
+            .classificationLabel(ClassificationLabel.REPORTED) // @TODO: logic behind this value to be defined
+            .dateReceipt(receipt.getCreationDate())
+            .dateReporting(installment.getNotificationDate())
+            .dateTreasury(receipt.getTransferDate())
```

Comment:
these fields will be filled by AssessmentClassificationWF, let's remove them here
```suggestion
```